common/src/main/java/com/viaversion/viaversion/protocols/protocol1_13to1_12_2/packets/WorldPackets.java

```diff
@@ -535,6 +537,32 @@ else if (particleId == 36)
                 });
             }
         });
+
+        // Incoming Packets
+        protocol.registerServerbound(ServerboundPackets1_13.PLAYER_BLOCK_PLACEMENT, wrapper -> {
+            Position pos;
+            wrapper.write(Type.POSITION, pos = wrapper.read(Type.POSITION)); // Location
+            wrapper.write(Type.VAR_INT, wrapper.read(Type.VAR_INT)); // block face
+            wrapper.write(Type.VAR_INT, wrapper.read(Type.VAR_INT)); // hand
+            wrapper.write(Type.FLOAT, wrapper.read(Type.FLOAT)); // cursor x
+            wrapper.write(Type.FLOAT, wrapper.read(Type.FLOAT)); // cursor y
+            wrapper.write(Type.FLOAT, wrapper.read(Type.FLOAT)); // cursor z
+
+            if (Via.getConfig().isServersideBlockConnections() && ConnectionData.needStoreBlocks()) {
+                ConnectionData.markModified(wrapper.user(), pos);
+            }
+        });
+        protocol.registerServerbound(ServerboundPackets1_13.PLAYER_DIGGING, wrapper -> {
+            Position pos;
+            wrapper.write(Type.VAR_INT, wrapper.read(Type.VAR_INT)); // Status
```

Comment:
Consider using map() instead of write(read()) like done in the SPAWN_PARTICLE handler